site/content/admin-guide/amqp-adapter-config.md

```diff
@@ -0,0 +1,212 @@
+| `HONO_AMQP_NATIVE_TLS_REQUIRED`<br>`--hono.amqp.nativeTlsRequired` | no | `false` | The server will probe for OpenSLL on startup if a secure port is configured. By default, the server will fall back to the JVM's default SSL engine if not available. However, if set to `true`, the server will fail to start at all in this case. |
+| `HONO_AMQP_PORT`<br>`--hono.amqp.port` | no | `8883` | The secure port that the protocol adapter should listen on.<br>See [Port Configuration]({{< relref "#port-configuration" >}}) below for details. |
+| `HONO_AMQP_SECURE_PROTOCOLS`<br>`--hono.amqp.secureProtocols` | no | `TLSv1.2` | A (comma separated) list of secure protocols that are supported when negotiating TLS sessions. Please refer to the [vert.x documentation](https://vertx.io/docs/vertx-core/java/#ssl) for a list of supported protocol names. |
+
+The variables only need to be set if the default values do not match your environment.
+
+
+## Port Configuration
+
+The AMQP protocol adapter can be configured to listen for connections on
+
+* a secure port only (default) or
+* an insecure port only or
+* both a secure and an insecure port (dual port configuration)
+
+The AMQP protocol adapter will fail to start if none of the ports is configured properly.
+
+### Secure Port Only
+
+The protocol adapter needs to be configured with a private key and certificate in order to open a TLS secured port.
+
+There are two alternative ways for doing so:
+
+1. either setting the `HONO_AMQP_KEY_STORE_PATH` and the `HONO_AMQP_KEY_STORE_PASSWORD` variables in order to load the key & certificate from a password protected key store, or
+1. setting the `HONO_AMQP_KEY_PATH` and `HONO_AMQP_CERT_PATH` variables in order to load the key and certificate from two separate PEM files in PKCS8 format.
+
+When starting up, the protocol adapter will bind a TLS secured socket to the default secure port 4041. The port number can also be set explicitly using the `HONO_AMQP_PORT` variable.
+
+The `HONO_AMQP_BIND_ADDRESS` variable can be used to specify the network interface that the port should be exposed on. By default the port is bound to the *loopback device* only, i.e. the port will only be accessible from the local host. Setting this variable to `0.0.0.0` will let the port being bound to **all** network interfaces (be careful not to expose the port unintentionally to the outside world).
+
+### Insecure Port Only
+
+The secure port will mostly be required for production scenarios. However, it might be desirable to expose a non-TLS secured port instead, e.g. for testing purposes. In any case, the non-secure port needs to be explicitly enabled either by
+
+- explicitly setting `HONO_AMQP_INSECURE_PORT` to a valid port number, or by
```

Comment:
there doesn't seem to be such a property ...

Reply:
when starting an insecure port, this property is used to determine which insecure port the server should bind to. The default is `4040` if this configurable property is not explicitly set.

Reply:
This property is not contained in the table above ...